app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -96,7 +96,7 @@ import { downloadCalendar } from "discourse/lib/download-calendar";
 // based on Semantic Versioning 2.0.0. Please up the changelog at
 // docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md whenever you change the version
 // using the format described at https://keepachangelog.com/en/1.0.0/.
-const PLUGIN_API_VERSION = "1.0.0";
+const PLUGIN_API_VERSION = "1.1.0";
```

Comment:
:grin: Changelog needs to be updated too

Reply:
Derp

Reply:
I am so careless